File: app/src/main/java/com/mb_nthw/landlord/MainActivity.java
package com.mb_nthw.landlord;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    ImageView uAddLandlord,
              uAddTenat;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        uAddLandlord = (ImageView) findViewById(R.id.add_landlord);

        uAddTenat = (ImageView) findViewById(R.id.add_tenant);

        uAddLandlord.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, LandlordRegistration.class);
                startActivity(intent);
            }
        });

        uAddTenat.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, LandlordRegistration.class);
                startActivity(intent);
            }
        });



    }
}

File: app/src/main/java/com/stavros/demo/android/form/MainActivity.kt
package com.stavros.demo.android.form

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.stavros.demo.android.R
import com.stavros.demo.android.util.FormType
import com.stavros.demo.android.util.startJsonFormActivity

class MainActivity : AppCompatActivity() {

    var uAddLandlord: ImageView? = null
    var uAddTenant: ImageView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setTitle(getString(R.string.agent_app))

        uAddLandlord = findViewById(R.id.add_landlord)
        uAddTenant = findViewById(R.id.add_tenant)

        uAddLandlord?.setOnClickListener {
            /*val intent = Intent(this@MainActivity, LandlordRegistration::class.java)
            startActivity(intent)*/
            var intent = startJsonFormActivity(this, "Landlord Registration", "landlord-registration.neat.json", LandlordRegistrationActivity::class.java)
            startActivity(intent)
        }

        uAddTenant?.setOnClickListener {
            var intent = startJsonFormActivity(this, "Tenant Registration", "tenant-registration.neat.json", TenantRegistrationActivity::class.java)
            startActivity(intent)

            /*val intent = Intent(this@MainActivity, LandlordListActivity::class.java)
            startActivity(intent)*/
        }
    }
}